setup.py

```diff
@@ -1,26 +1,35 @@
-from setuptools import setup, find_namespace_packages
+import os
+import sys
+import pathlib
+import setuptools
 
 install_requires = ["numpy"]
-tests_require = ["pytest", "pytest-cov", "pytest-flake8"]
+tests_require = ["pytest", "pytest-cov", "pytest-flake8", "dds"]
```

Comment:
Surely `tests_require` is extra requirements needed to run unit tests (as opposed to just using the package). But "dds" is needed to *use* the package, so shouldn't it be listed somewhere else, such as `install_requires`? Also "astropy" seems to be missing and it is in the same category as "dds".